sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -365,18 +365,20 @@ class HiveCatalogedDDLSuite extends DDLSuite with TestHiveSingleton with BeforeA
 
   test("Database Ownership") {
     val catalog = spark.sessionState.catalog
-    try {
-      val db = "spark_29425_1"
-      sql(s"CREATE DATABASE $db")
-      assert(sql(s"DESCRIBE DATABASE EXTENDED $db")
-        .where("database_description_item='Owner'")
```

Comment:
can we use the new column name instead of turning on the legacy config?

Reply:
> can we use the new column name instead of turning on the legacy config?

TBH, I am confused why we use such a strange col name before, can we just ignore this?

Reply:
> can we use the new column name instead of turning on the legacy config?

Updated